forms.rst

```diff
@@ -560,46 +560,51 @@ To see the second approach - adding constraints to the form - and to
 learn more about the validation constraints, please refer to the
 :doc:`Symfony validation documentation </validation>`.
 
+Friendlier validation messages
```

Comment:
In the docs we use a "title case", where all words are capitalized except from closed-class words.

I would also propose a more neutral title like "Form Validation Messages"